### PR TITLE
Fix LoRA test by handling mask creation inside the test

### PR DESCRIPTION
**Pull request by SanjuCSudhakaran**
This PR handles mask creation inside lora unit tests to align with new BGMV implementation

### CODE REVIEW

File: tests/lora/test_lora_hpu.py [hunk 1]
```diff
@@ -39,32 +52,40 @@ def test_apply_lora(m, n, k, rank, dtype) -> None:
     input = torch.rand(k, n, device="hpu", dtype=dtype)
     expected = input @ lora.lora_a @ lora.lora_b * lora.scaling
 
-    lora_a_stack = torch.zeros(MAX_LORAS + 1,
+    lora_a_stack = torch.zeros(8,
```
Comment (hlahkar) [inline tests/lora/test_lora_hpu.py:55]: any reason why we removed the variable and hardcoded it here

**Review by hlahkar — APPROVED** [review]

**Review by vivekgoe — APPROVED** [review]
